lib/transformer.dart

```diff
@@ -59,7 +60,25 @@ TransformOptions _parseSettings(Map args) {
       htmlFiles: _readStringListValue(args, 'html_files'),
       sdkDirectory: sdkDir,
       templateUriRewrites: _readStringMapValue(args, 'template_uri_rewrites'),
-      diOptions: diOptions);
+      diOptions: diOptions,
+      generateTemplateCache: _readBoolValue(args, 'generate_template_cache'));
+}
+
+_readBoolValue(Map args, String name, {bool required: true}) {
```

Comment:
why it is required. I would assume it is optional, at least `generate_template_cache` should be.